state/state.yml

```diff
@@ -114,3 +114,78 @@ properties:
             blocked-interface:
               type: string
               description: The interface (e.g., port) where the blocked entry was detected.
+  stp:
+    description: Global status and configuration of the Spanning Tree Protocol (STP) across the switch.
+      Supports variants like STP, RSTP, MSTP. Includes bridge-wide parameters and protocol behavior.
+    type: object
+    properties:
+      enabled:
+        type: boolean
+        description: Whether STP is enabled globally on the switch.
+      mode:
+        type: string
+        description: Operational STP mode used across the switch.
+        enum:
+          - STP
+          - RSTP
+          - MSTP
+          - PVST
+          - RPVSTP
+      transmission-limit:
+          type: integer
+          description: Maximum number of BPDUs that can be transmitted per hello time.
+      path-cost-method:
+          type: string
+          description: Indicates whether "Short" or "Long" path cost method is in use.
+          enum:
+            - Short
+            - Long
+      max-hops:
+          type: integer
+          description: Maximum hop count used in MSTP.
+      instances:
+      type: array
+      description: List of STP instances (e.g., MSTP instances).
```

Comment:
correct indentation for instances property, it fails json schema validation, creates a null property for instances, etc.